src/parser/metrics/RealLinesOfCode.ts

```diff
@@ -72,12 +74,17 @@ export class RealLinesOfCode implements Metric {
 
     async calculate(parsedFile: ParsedFile): Promise<MetricResult> {
         const { language, tree } = parsedFile;
-        let isCommentFunction: (node: Parser.SyntaxNode) => boolean;
-
-        if (language == Language.Python) {
-            isCommentFunction = (node: Parser.SyntaxNode) => this.isPythonComment(node);
-        } else {
-            isCommentFunction = (node: Parser.SyntaxNode) => this.isComment(node);
+        let isCommentFunction: (node: SyntaxNode) => boolean = (node: SyntaxNode) =>
+            this.isComment(node);
+        let countAllLinesFunction: (node: SyntaxNode) => boolean = isLeafNodeButNoLinebreak;
+
+        switch (language) {
+            case Language.Python:
+                isCommentFunction = (node: SyntaxNode) => this.isPythonComment(node);
+                break;
+            case Language.Bash:
```

Comment:
Ok, no, it is not, because that special handling is caused by an issue of the tree-sitter-bash grammar. But as it is a rather minor special handling, we probably keep it. 

I have added documentation to the special handling to clearify the reason for it.